### ISSUE TITLE
6.0beta07 broke support of static-challenge-response.user.sh hook

### ISSUE THREAD

**Issue by igorlidin** [Tunnelblick/Tunnelblick#821]
**Describe the bug**
6.0beta07 calls the static-challenge-response.user.sh hook no more

**To Reproduce**
1. add static-challenge variable to config.ovpn of a profile
2. add static-challenge-response.user.sh hook to a profile
3. initiate connection of a profile
4. the hook is not called

**Expected behavior**
static-challenge-response.user.sh should be called if exists and if static-challenge variable is set in a config

**Comment by jkbullard** [comment]
@igorlidin -  Please post the diagnostic info obtained by following the instructions at [Read Before You Post](https://tunnelblick.net/cBeforeYouPost.html).

**Comment by igorlidin** [comment]
6.0beta06 works, 6.0beta07 fails to call a hook, 6.0beta08 still broken.

**Comment by igorlidin** [comment]
> @igorlidin - Please post the diagnostic info obtained by following the instructions at [Read Before You Post](https://tunnelblick.net/cBeforeYouPost.html).

Sorry, way too much of sensitive data in a diagnostics. If you would like me to provide some specific non-sensitive info, you are always welcome.

**Comment by jkbullard** [comment]
@igorlidin - Please remove what you consider sensitive and post that.

**Comment by csoroiu** [comment]
the idea is that instead of calling the `static-challenge-response.user.sh` hook, the above versions are prompting you for the challenge:
<img width="612" alt="image" src="https://github.com/user-attachments/assets/6f00a3e9-9502-43c5-8190-5a2eafb8c7a3">

**Comment by csoroiu** [comment]
Right, but before upgrading to 6.0beta07 i didn't got the above dialog unless I didn't had the password saved.
And right after upgrade i get it every time and static-challenge-response.user.sh is not called anymore.

Basically scenario is:
1. Tunelblick < 6.0beta07 configured with openvpn config + static-challenge-response.user.sh is connecting automatically
2. upgrade tunnelblick, you get the above dialog at every connect.

Thanks, hope this helps.

**Comment by igorlidin** [comment]
@jkbullard Sorry for bothering, any chance to fix this? I have tried to figure out how to fix it myself, but no success. It seems for me that in last commits the code which called the function that called the hook is just removed.

**Comment by igorlidin** [comment]
Still broken in 6.0beta10 — tunnelblick does not call static-challenge-response.user.sh, as 6.0beta07 does.

**Comment by jkbullard** [comment]
Tunnelblick 6.0beta11 should fix this problem.
